SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/SpatialGDKEditorSchemaGeneratorTest.cpp

```diff
@@ -1225,4 +1227,135 @@ SCHEMA_GENERATOR_TEST(GIVEN_actor_class_WHEN_generating_schema_THEN_expected_com
 	return true;
 }
 
+SCHEMA_GENERATOR_TEST(
+	GIVEN_snapshot_affecting_schema_files_WHEN_hash_of_file_contents_is_generated_THEN_hash_matches_expected_snapshot_version_hash)
+{
+	SchemaTestFixture Fixture;
+
+	// GIVEN
+	FString GDKSchemaCopyDir = FPaths::Combine(SpatialGDKServicesConstants::SpatialOSDirectory, TEXT("schema/unreal/gdk"));
```

Comment:
I see `"schema/unreal/gdk"` used a few times throughout the codebase, may be worth refactoring into its own constant somewhere

Reply:
Good idea, best to tackle in a separate PR, could you add a task for this?

Reply:
https://improbableio.atlassian.net/browse/UNR-5129